presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java

```diff
@@ -45,7 +46,10 @@
         @Override
         public Requirement getRequirements(Configuration configuration)
         {
-            return mutableTable(NATION);
+            return mutableTable(
+                    HiveTableDefinition.from(NATION)
+                            .injectStats(false)
```

Comment:
is this what was added in the new version of tempto?

Reply:
yes -- the ability to control injecting stats on per-table basis, overriding global `inject_stats_for_mutable_tables`